system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -495,13 +495,18 @@
  ([provider-id request-body options]
   (let [accept-format (get options :accept-format :xml)
         token (:token options)
+        user-id (:user-id options)
+        headers (when (or user-id token)
+                  (util/remove-nil-keys
+                    {transmit-config/token-header token
+                     "user-id" user-id}))
```

Comment:
Even though the keys here are "echo-token" and "user-id", but when it gets to handle-bulk-update-event in the bulk_update_service.clj,  these keys are changed to :echo-token and :user-id.   Not sure how it happened. I can't modify api/core  set-user-id to retrieve these using the :echo-token :user-id because it will break existing tests.